DS/DS8/SortingSystem.java

```diff
@@ -0,0 +1,92 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+public class SortingSystem {
+	static int[] list=null;
+	public static void input() {
+		Scanner sc=new Scanner(System.in);
+		int sizeOfList=0;
+		boolean isPositive=false;
+		try {
+			while(!isPositive) {
+				System.out.println("Enter size of array");
+				sizeOfList=sc.nextInt();
+				if(sizeOfList<=0) {
+					System.out.println("Enter only positive value");
+				}
+				else {
+					isPositive=true;
+				}
+			}
+		list=new int[sizeOfList];
+		System.out.println("Enter number in list:");
+		TreeSorting.addElement(list,sc);
```

Comment:
ACTION: Why are you using addElement method of  class treesorting here.addElement() function is not specific to TreeSorting class, This method is to add elements in a array then it should be defined inside main clsss(SortingSystem).